reporter/src/main/kotlin/reporters/ReportTableModel.kt

```diff
@@ -211,13 +211,20 @@ data class ReportTableModel(
     data class IssueTable(
         val rows: List<IssueRow>
     ) {
-        val unresolvedIssues = rows.flatMap {
-            it.analyzerIssues.flatMap { (_, issues) -> issues } + it.scanIssues.flatMap { (_, issues) -> issues }
-        }.filterNot { it.isResolved }.groupBy { it.severity }
-
-        val errorCount = unresolvedIssues[Severity.ERROR].orEmpty().size
-        val warningCount = unresolvedIssues[Severity.WARNING].orEmpty().size
-        val hintCount = unresolvedIssues[Severity.HINT].orEmpty().size
+        val errorCount: Int
+        val warningCount: Int
+        val hintCount: Int
+
+        init {
+            val unresolvedIssues = rows.flatMap {
+                it.analyzerIssues.flatMap { (_, issues) -> issues } +
+                        it.scanIssues.flatMap { (_, issues) -> issues }
```

Comment:
Nit: This still would have fitted into the previous line.

Reply:
Are you aware that this is already merged? If so, what's your intention behind commenting nits on already merged PRs? I'm not quite sure how to interpret these comments.

Reply:
> Are you aware that this is already merged?

Yes, that's why I wrote "would have fitted" in past tense.

> If so, what's your intention behind commenting nits on already merged PRs?

The ask to watch more closely for these kinds of (minor) formatting issues in future PRs.

Reply:
Thanks for explaining.